reflectapi/src/codegen/rust-dependency-stubs/http.rs

```diff
@@ -17,3 +18,56 @@ impl StatusCode {
         unimplemented!()
     }
 }
+
+#[derive(Clone)]
+pub struct HeaderMap<T = HeaderValue> {
+    _phantom: core::marker::PhantomData<T>,
+}
+
+impl<T> HeaderMap<T> {
+    pub fn new() -> Self {
+        unimplemented!()
+    }
+
+    pub fn insert(&mut self, _name: HeaderName, _value: T) {
+        unimplemented!()
+    }
+}
+
+pub struct HeaderName(());
+
+impl HeaderName {
+    pub fn from_bytes(_bytes: &[u8]) -> Result<Self, InvalidHeaderName> {
+        unimplemented!()
```

Comment:
it is not real code? only stubs?

Reply:
Yeah, it's a bit of a hack to do typechecking to ensure the generated rust code is correct for tests at least. I don't mind removing it but it's pretty helpful.